benchexec/tablegenerator/react-table/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is this related to the changes here? If not, I suggest to remove it (and add it separately if desired).

Reply:
I added it at the same time as the new `test:debug` script in the package.json. 
I can commit them to master separately though if you would prefer that.

Reply:
Seems my upvote was not noticed. Yes, please, separate these changes.